apps/playground/app/test-text-wrap/page.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import {
+  Theme,
+  Flex,
+  Grid,
+  TextFieldInput,
+  Text,
+  SelectRoot,
+  SelectTrigger,
+  SelectContent,
+  SelectItem,
+  TextArea,
+  Button,
+  Container,
+  Section,
+  Card,
+  IconButton,
+  Heading,
+  Code,
+  ScrollArea,
+} from '@radix-ui/themes';
+import { NextThemeProvider } from '../next-theme-provider';
+
+export default function Test() {
+  return (
+    <html lang="en" suppressHydrationWarning>
+      <body>
+        <NextThemeProvider>
+          <Theme asChild>
+            <div id="root">
+              <Container px="8">
+                <Section size="3">
+                  <Grid columns="3" gap="5">
+                    <Card size="3">
+                      <ScrollArea>
+                        <Heading size="3" mb="2">
+                          Default
+                        </Heading>
+                        <Text size="3" as="p">
+                          Under a sky as clear as a sapphire, the audacious wind embarked on yet
+                          another sojourn.
+                        </Text>
+                      </ScrollArea>
+                    </Card>
+
+                    <Card size="3">
+                      <Heading size="3" mb="2">
+                        <Code variant="ghost" color="gray" highContrast>
```

Comment:
The ghost Code color gotcha [got me again](https://github.com/workos/workos/blob/main/packages/design-system/src/overrides.css#L1-L8). I'll take care of it. @benoitgrelard remind me, did we agree we can do it without a major release? I reckon we talked about it.